README.md

```diff
@@ -121,7 +121,11 @@ jobs:
 | `set-java-home` | `'true'` | If set to `'true'`, instructs the action to set `$JAVA_HOME` to the path of the GraalVM installation. |
```

Comment:
What if there's more than one GraalVM installation?

Reply:
I believe the last run wins. So far, however, we haven't seen any use cases that require two different GraalVM installations.

Reply:
In which case, I think it would be good practice to report to the user the new value of $JAVA_HOME. Or at least log it somewhere.

Reply:
Right, GitHub Actions supports that by [enabling debug logging](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging). Then, this line will trigger some additional logging and print the path to `JAVA_HOME`: https://github.com/graalvm/setup-graalvm/blob/c2c2a821994792cab6df1f0f05eec4bf9442db7d/src/main.ts#L65

AFAIKT, the official [`setup-java`](https://github.com/actions/setup-java) action doesn't explicitly print the path either.